testsuite/tests/contrib/lkt_semantic/cast/test.lkt

```diff
@@ -1,4 +1,6 @@
-class RootNode : Node {}
+@abstract class CastNode {}
+
+class RootNode : CastNode {}
```

Comment:
We don’t check this right now but the root node should implement Node all other classes should derive from it: could you fix accordingly?